src/codeflare_sdk/cluster/cluster.py

```diff
@@ -456,6 +415,29 @@ def job_logs(self, job_id: str) -> str:
         """
         return self.job_client.get_job_logs(job_id)
 
+    @staticmethod
+    def _head_worker_extended_resources_from_rc_dict(rc: Dict) -> Tuple[dict, dict]:
+        head_extended_resources, worker_extended_resources = {}, {}
+        for resource in rc["spec"]["workerGroupSpecs"][0]["template"]["spec"][
+            "containers"
+        ][0]["resources"]["limits"].keys():
```

Comment:
Just curious, is it intentional to only processes the first container and extract custom extended resources?

Reply:
Yeah, the extended resources will only apply to the container running the actual Ray runtime. The code naively assumes that this will always be the first listed container